fiaas_skipper/deploy/deploy.py

```diff
@@ -47,9 +47,15 @@ def _deploy(self, deployment_config, channel):
 
     @staticmethod
     def _create_metadata(deployment_config):
+        labels = {
+            "app": deployment_config.name,
+            "heritage": "Skipper",
```

Comment:
Not sure but wonder if we want to include the fiaas prefix in there ie fiaas-skipper?

Reply:
Fair point I guess. I just used Skipper because helm uses "Tiller", but we can be less arrogant ;)